src/api/GraphQuery/queryBuilder.ts

```diff
@@ -211,12 +211,12 @@ export interface BaseStrictPropertyFilter<T> extends BasePropertyFilter {
 
 export interface StringStrictPropertyFilter extends BaseStrictPropertyFilter<string[]> {
   propertyType: 'string';
-  operator: '=' | '!=';
+  operator: '=' | '!=' | 'null' | 'notNull';
```

Comment:
suggestion: using « isNull » and « isNotNull », because « null » looks like someone casted `null` into a string by mistake.